src/frontend/src/icp/components/send/IcSendAmount.svelte

```diff
@@ -27,85 +31,70 @@
 	let fee: bigint | undefined;
 	$: fee = ($token as IcToken).fee;
 
-	const validate = () => {
-		if (invalidAmount(amount)) {
-			amountError = undefined;
-			return;
-		}
-
+	$: customValidate = (userAmount: BigNumber) => {
```

Comment:
Should we provide the return type here? If yes, in another PR given that it's the same omission in `SendAmount.svelte`

```
$: customValidate = (userAmount: BigNumber): Error | undefined => {
```

Reply:
right! will do